eo-parser/src/main/resources/org/eolang/parser/move-versions-deeper.xsl

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License (MIT)
+
+Copyright (c) 2016-2023 Objectionary.com
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" id="move-versions-deeper" version="2.0">
+  <!--
+  Here we move "ver" attribute from the top object deeper to pre-last one (before <o base="Q"/>)
+
+  1. The reason of transformation - move version to pre-last object ("org"). Then on "transpile"
```

Comment:
@volodya-lombrozo I believe it would be harder and slower to implement, because we will have to manipulate what is not yet in any form, even if not quite right. 
When you don't know for sure how this whole feature should work, you can't just come and say "we need this here, this there, and this out there". You need to start with something small, maybe not quite elegant, just to have something in front of you so you can work with it

Reply:
@maxonfjvipon Exactly:
> You need to start with something small, maybe not quite elegant, just to have something in front of you so you can work with it

But you don't start your work with something random. The first thing you usually do is answer the question, "what do I want to achieve?" and then draw a skeleton. After that, you start adding "meat" to that skeleton. When we have a skeleton and an endpoint, we don't ask questions like "why do we need that change?" because it's obvious.
Otherwise, you'll find yourself in the situation we're currently facing: no clear reason for the change and no understanding of the final goal we aim to achieve. The next day, any developer could easily remove those changes, and they'd be justified in doing so.

Suddenly, I didn't find a good article how to do so, but may be [this](https://www.yegor256.com/2010/03/04/pdd.html) blog post might explain the idea a bit:

>  Let’s keep all these unknowns in mind as we try to solve the problem on the highest level of abstraction. Of course, we start with a test